### ISSUE TITLE
Share links to folders have extra whitespace before file list due to div#rich-workspace

### ISSUE THREAD

**Issue by emichael** [nextcloud/server#21616]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Create a read-only share link to a folder
2. Visit that folder, either from a logged in window or an incognito one

### Expected behaviour
There shouldn't be extra whitespace above the file list.

### Actual behaviour
There is extra whitespace above the file list.

![image](https://user-images.githubusercontent.com/1312141/85953296-4113cf80-b924-11ea-923a-bf6bca33acd5.png)

This is caused by the `#rich-workspace` div having `min-height: 90px;`. I can't seem to find where this happens with some cursory GitHub searches.

For now, I've added some custom CSS to fix the issue.

```
#body-public #content.app-files_sharing #files-public-content #filelist-header #rich-workspace {
  display: none;
}
```

### Server configuration

**Operating system:**
Ubuntu Server 18.04

**Nextcloud version:** 19.0.0 (snap latest)

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - accessibility: 1.5.0
  - activity: 2.12.0
  - breezedark: 19.0.2
  - bruteforcesettings: 1.6.0
  - cloud_federation_api: 1.2.0
  - dav: 1.15.0
  - federatedfilesharing: 1.9.0
  - federation: 1.9.0
  - files: 1.14.0
  - files_pdfviewer: 1.8.0
  - files_rightclick: 0.16.0
  - files_sharing: 1.11.0
  - files_trashbin: 1.9.0
  - files_versions: 1.12.0
  - files_videoplayer: 1.8.0
  - logreader: 2.4.0
  - lookup_server_connector: 1.7.0
  - news: 14.1.11
  - nextcloud_announcements: 1.8.0
  - notifications: 2.7.0
  - oauth2: 1.7.0
  - photos: 1.1.0
  - provisioning_api: 1.9.0
  - recommendations: 0.7.0
  - serverinfo: 1.9.0
  - settings: 1.1.0
  - sharebymail: 1.9.0
  - text: 3.0.1
  - theming: 1.10.0
  - theming_customcss: 1.6.0
  - twofactor_backupcodes: 1.8.0
  - twofactor_totp: 4.1.3
  - viewer: 1.3.0
  - workflowengine: 2.1.0
Disabled:
  - admin_audit
  - comments
  - contactsinteraction
  - encryption
  - files_external
  - firstrunwizard
  - password_policy
  - privacy
  - support
  - survey_client
  - systemtags
  - user_ldap
```
</details>

### Client configuration
**Browser:** Chrome latest

**Operating system:** Ubuntu

**Comment by violoncelloCH** [comment]
@juliushaertl could this whole section be hidden in case `Readme.md` is empty and there is no write permission?

**Comment by juliushaertl** [comment]
This will be fixed with the upcoming 19.0.1 already.
